date/date_c.php

```diff
@@ -8,62 +8,62 @@ interface DateTimeInterface {
      * @since 7.2
      */
     const ATOM = 'Y-m-d\TH:i:sP';
-    
+
```

Comment:
Oops, PHPStorm fixed some empty lines when saving the file :)
Leaving as is though, I think this deserves to be fixed, too.